class/Abstracts/Trigger.php

```diff
@@ -11,7 +11,7 @@
 use BracketSpace\Notification\Interfaces\Sendable;
 use BracketSpace\Notification\Admin\FieldsResolver;
 use BracketSpace\Notification\Defaults\Store\Notification as NotificationStore;
-use BracketSpace\Notification\Core\Notification;
+use BracketSpace\Notification\Core\Notification as NotificationCore;
```

Comment:
I don't like that name as it suggests that Notifications have some core when they not.

```suggestion
use BracketSpace\Notification\Core\Notification as CoreNotification;
```